CHANGELOG.md

```diff
@@ -24,6 +24,16 @@ so they can be moved to a respective version upon release.
 
 Add your changes here.
 
+## [2.1.0] - 2020-02-24
+
+- Fine tune API changes ([#216](https://github.com/airsidemobile/JOSESwift/pull/216)) via [@daniel-mohemian](https://github.com/daniel-mohemian)
+- Adds: A128KW, A192KW, and A256KW algorithms ([#211](https://github.com/airsidemobile/JOSESwift/pull/211)) via [@mtamu](https://github.com/mtamu)
+- Changes JWE encryption and decryption flow ([#210](https://github.com/airsidemobile/JOSESwift/pull/210)) via [@daniel-mohemian](https://github.com/daniel-mohemian)
```

Comment:
Same here, what does "Changes JWE encryption and decryption flow" mean for the end user? Should also probably be past tense.